days/src/day10/Day10.kt

```diff
@@ -0,0 +1,31 @@
+package day10
+
+import Day
+import java.io.File
+
+class Day10(values: List<Int>) : Day {
+    private val jolts = values.sorted().let { listOf(0) + it + it.last() }
```

Comment:
Sorting first was smart, I just build a range and scanned for candidates each time